docs/dev/fundamentals/zkSync.md

```diff
@@ -0,0 +1,72 @@
+Thanks to the combination of cutting-edge cryptography and on-chain data availability, zkRollup (the core network of zkSync) is the only L2 scaling solution that doesn't require any operational activity to keep the funds safe. 
+For example, users can go offline and still be able to withdraw their assets safely when they come back, even if the zkRollup validators are no longer around.
+
+In the following sections we will cover some attributes of zkSync.
+
+## Transaction types
+
+zkSync supports Ethereum's "legacy" (pre-EIP2718) transaction types, EIP1559 transaction type, and its custom EIP712 transactions. You can use transactions of this type to use zkSync-specific features like account abstraction. 
+Additionally, it is only possible to deploy smart contracts with this type of transaction.
+
+Knowing the details about the transaction format is not required to use zkSync's SDK, but if you are curious, you can read more about it [here](../../api/api.md#eip712).
+
+
+## Confirmations and finality
+
+On zkSync, each transaction is at one of the four stages:
+
+- `Pending`: The transaction was received by the operator, but it has not been processed yet.
+- `Processed`: The transaction is processed by the operator and is confirmed to be included in the next block.
+- `Committed`: The transaction state diffs were published on Ethereum.
+- `Finalized`: The SNARK validity proof for the transaction has been submitted and verified by the smart contract. After this step, the transaction is considered to be final.
+
+The typical time for a transaction to go from `Processed` to `Finalized` is a couple of hours at the current stage.
+
+Please note that for developer convenience, we usually treat the `Processed` and `Committed` states as a single stage called `Committed` since they have no difference from the UX/DexEx standpoints.
+
+
+## zkSync features
+
+- ETH and ERC20 token transfers with instant confirmations and 10-minute finality on L1
+- Transaction fees are extremely low (1/100th of the mainnet cost for ERC20 tokens and 1/30th for ETH transfers).
+- No registration is required to receive funds.
+- Payments to existing Ethereum addresses (including smart contracts) can be conveniently paid with the token being transferred.
+- Withdrawals to the mainnet take ~10 minutes.
+- As with the Ethereum mainnet, no third parties are needed to keep funds safe.
+- Users are always in control of their funds.
+- zkSync is smart-contract friendly; it allows you to reuse existing Solidity code.
+- zkSync is friendlier for exchanges.
+- zkSync has native support for NFTs.
+- There is no requirement for an operational activity to keep the funds safe.
+
+### zkSync 2.0 highlights
+
+- Mainnet-like security with zero reliance on 3rd parties.
+- Permissionless EVM-compatible smart contracts.
+- Standard Web3 API.
+- Preserving key EVM features, such as smart contract composability.
+- Introducing new features, such as account abstraction and meta-transactions.
```

Comment:
```suggestion
- Introducing new features, such as account abstraction.
```

We don't have meta-transactions per se, the users could implement something like them if they wish to via account abstraction.